index.js

```diff
@@ -3222,6 +3240,27 @@ ModuleStore.prototype.getProvidersForModule = function (moduleName) {
 									enumeration: integerEnumeration
 								};
 							}
+							// If this column has syntax RowStatus and
+							// the MIB module imports RowStatus from
+							// SNMPv2-TC, mark this column as the
+							// rowStatus column so we can act on it.
+							// (See lib/mibs/SNMPv2-TC.mib#L186.)
+							if ( syntax == "RowStatus" &&
+									"IMPORTS" in mibModule &&
+									Array.isArray(mibModule.IMPORTS["SNMPv2-TC"]) && 
+									mibModule.IMPORTS["SNMPv2-TC"].includes("RowStatus") ) {
+
+								// Mark this column as being rowStatus
+								columnDefinition.rowStatus = true;
+
+								// Also keep track of the rowStatus
+								// column on a provider basis. (Note
+								// that this will not work correctly
+								// if there are multiple columns in a
+								// single table with syntax RowStatus.
+								// Only the last one will be honored.)
```

Comment:
That's fine - the intention is to have a single RowStatus column per table, so this is the only case we need to cater for..  From RFC 2578 Section 7.1.12.1:

"For newly-defined conceptual rows which allow the creation of new object instances and/or the deletion of existing object instances, there should be one columnar object with a SYNTAX clause value of RowStatus."